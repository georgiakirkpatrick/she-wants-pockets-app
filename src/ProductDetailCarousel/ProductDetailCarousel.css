.ProductDetailCarousel {
    width: 100%;
    overflow: scroll;
}

.ProductDetailCarousel__items {
    display: flex;
    flex-direction: row;
    overflow: scroll;
}

.ProductDetailCarousel__items > img {
    width: 80vw;
    padding: 0 10px 0 0;
}

.ProductDetailCarousel__items > div {
    padding: 0 10px 0 0;
    width: 300px;
}

#section-carousel > .ProductDetailCarousel__items {
    width: 150%;
}