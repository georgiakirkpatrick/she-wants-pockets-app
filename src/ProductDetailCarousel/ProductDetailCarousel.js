import React from 'react'
import './ProductDetailCarousel.css'

const ProductDetailCarousel = props => {
    return (
        <div 
            id={props.id}
            className='ProductDetailCarousel'
        >
            <div className='ProductDetailCarousel__items'>
                {props.children}
            </div>
        </div>
    )
}

ProductDetailCarousel.defaultProps = {
    id: 0
}

export default ProductDetailCarousel