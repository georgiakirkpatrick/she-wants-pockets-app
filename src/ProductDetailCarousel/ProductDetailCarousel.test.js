import React from 'react'
import ReactDOM from 'react-dom'
import ProductDetailCarousel from './ProductDetailCarousel'
import renderer from 'react-test-renderer'

describe ('ProductDetailCarousel', () => {
    it ('renders without crashing', () => {
        const div = document.createElement('div')
        ReactDOM.render(<ProductDetailCarousel />, div)
        ReactDOM.unmountComponentAtNode(div)
    })

    it('renders the UI as expected', () => {
        const tree = renderer
            .create(<ProductDetailCarousel />)
            .toJSON()
        expect(tree).toMatchSnapshot()    
    })
})