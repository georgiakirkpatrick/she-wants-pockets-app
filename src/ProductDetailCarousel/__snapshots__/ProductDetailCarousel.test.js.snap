// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductDetailCarousel renders the UI as expected 1`] = `
<div
  className="ProductDetailCarousel"
  id={0}
>
  <div
    className="ProductDetailCarousel__items"
  />
</div>
`;
